### ISSUE TITLE
Dynamic Publishing / Reading From Disk

### ISSUE THREAD

**Issue by goldbattle** [irapkaist/file_player#2]
It would be nice if only certain topics are subscribed to then only those topics are published. For example if only the LiDAR is used, then there is no need to read the stereo images / imu / etc from disk. This can allow for faster dataset playback if not using all the sensors.

The main use case I have is if I am only using stereo + inertial, then I rather not have the system reading all the pointclouds and taking up *disk bandwidth*. The number of nodes that are currently subscribed to a topic can be checked with:
```
// Check if we have subscribers, if we don't then return
if(pub_odomimu.getNumSubscribers()==0)
     return;
```

**Comment by JinyongJeong** [comment]
Yes. I agree with your comments. Can you make a PR with what you said? If not, I will update the content later.
